repo_resource/common.py

```diff
@@ -96,14 +96,16 @@ def getRevision(remote, remoteUrl, project, branch):
     """
     # v1.0^{} is the commit referring to tag v1.0
     # git ls-remote returns the tag sha1 if left as is
+    branchOrig = branch
```

Comment:
Please find a better variable name.
`branchOrig` honestly does not means much.

I think I understand the problem but I don't think the current code is clear enough to explicit this issue.

Reply:
I've changes the logic in this commit:
https://github.com/makohoek/repo-resource/pull/40/commits/ec10c92a0764ffe685dbe03e21120c976a9958ec

`branchOrig` is not present anymore

Reply:
Looks better, thank you.